### DISCUSSION TITLE
How can I request a git tag (release) on nvim-lspconfig?

### DISCUSSION THREAD

**Discussion by christopher-francisco** [neovim/neovim#28565] [Q&A] [answered]
In order to use nvm-lspconfig at work I need to specify the version number I'll use (can't use git hashes). The last tag I see was 4 months ago. [README](https://github.com/neovim/nvim-lspconfig?tab=readme-ov-file#release-process) suggests I should just push a tag myself but I have no write access.

Any guidance is appreciated!

**Comment by clason** [comment]
That part of the docs is not meant for you...

You can fork it, of course, then you can tag to your heart's content.

**Reply by christopher-francisco** [reply]
I guess the question is more like is tagging/releasing still happening, or have the maintainers moved away from it?

**Reply by clason** [reply, marked as answer]
As this plugin is basically only a collection of configurations, there's no meaningful change that is driving tags or releases. So, basically, don't expect a proper release schedule.

**Reply by christopher-francisco** [reply]
That's alright, I was thinking that may be the case. I had to ask anyway. Thank you
